try.html

```diff
@@ -39,6 +39,18 @@
       alt: Python logo - Launch Jupyter Notebook demo Binder
       url: https://jupyter.org/try-jupyter/retro/notebooks/?path=notebooks/Intro.ipynb
 
+    - title: JupyterLite on GitHub Pages
+      description: JupyterLite (Wasm powered Jupyter) deployed as static GitHub Pages
+      src: try/jupyterlite.png
+      alt: JupyterLite logo - Launch JupyteLite demo Binder
+      url: https://jupyterlite.github.io/demo
+
+    - title: Jupyter Widgets
+      description: Interactive HTML widgets for Jupyter notebooks
```

Comment:
wait, working on it.

```suggestion
      description: Interactive HTML widgets for Jupyter notebooks
```